<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowNew</class>
 <widget class="QMainWindow" name="MainWindowNew">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>291</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" colspan="2">
      <widget class="QPushButton" name="pushButtonPokaz">
       <property name="text">
        <string>Pokaz</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButtonConnect">
       <property name="text">
        <string>Połącz</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QTableView" name="tableView_3">
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox"/>
     </item>
    </layout>
    <zorder>pushButtonConnect</zorder>
    <zorder>tableView_3</zorder>
    <zorder>comboBox</zorder>
    <zorder>pushButtonPokaz</zorder>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>50</y>
      <width>211</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Database driver:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radio_postgress">
       <property name="text">
        <string>Postgress</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radio_sqlite">
       <property name="text">
        <string>Sqlite3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>server:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditServer">
       <property name="text">
        <string>localhost/SQLITE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditDatabase">
       <property name="text">
        <string>people.db</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Plik bazy</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
